content/docs/user-guide/packaging/content-patch-package.md

```diff
@@ -0,0 +1,220 @@
+20. In the new file dialog, enter `game_v1_to_v2_patch_pc.assetlist` for the name, and choose **Open**.
+21. In the **Generate Asset List Files** dialog, choose **pc** from the platform list.
+22. Choose **Create New File**.
+
+{{% /tab %}}
+{{% tab name="Asset Bundler command" %}}
+
+Alternatively, you can run this command to generate the patch asset list.
+
+```cmd
+AssetBundlerBatch.exe compare --comparisonType delta ^
+    --firstAssetFile game_v1_pc.assetlist ^
+    --secondAssetFile game_v2_pc.assetlist ^
+    --output game_v1_to_v2_patch.assetlist
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+After a short time, a dialog notifies you that the asset list has been created successfully. You can verify that the patch asset list contains only the new changes by choosing the **Asset List** tab, selecting `game_v1_to_v2_patch` from **Asset list files**, and examining the contents in the main list window.
+
+### Create asset bundles
+
+Now you must create asset bundles (.pak files) for the new asset lists.
+
+{{< tabs name="Create asset bundles" >}}
+{{% tab name="Asset Bundler GUI" %}}
+
+To generate the patched release asset bundle, do the following:
+
+23. In the **Asset Lists** tab, select the `game_v2` asset list.
+24. Choose **Generate Bundle**.
+25. In the **Generate Bundles** dialog, choose **Browse...**.
+26. In the file dialog, enter `game_v2_pc.pak` for the file name and choose **Open**.
+27. Choose **Generate Bundles**.
+
+To generate the patch asset bundle, do the following:
+
+28. In the **Asset Lists** tab, select the `game_v1_to_v2_patch` asset list.
+29. Choose **Generate Bundle**.
+30. In the **Generate Bundles** dialog, choose **Browse...**.
+31. In the file dialog, enter `game_v1_to_v2_patch_pc.pak` for the file name and choose **Open**.
+32. Choose **Generate Bundles**.
+
+{{% /tab %}}
+{{% tab name="Asset Bundler command" %}}
+
+Alternatively, you you can run these commands to generate bundles from the asset lists.
+
+Use the following command to generate the patched release bundle containing all the game assets:
+
+```cmd
+AssetBundlerBatch.exe bundles --assetListFile game_v2_pc.assetlist ^
+    --outputBundlePath game_v2.pak
+```
+
+Use the following command to generate the patch bundle containing only the changed assets:
+
+```cmd
+AssetBundlerBatch.exe bundles --assetListFile game_v1_to_v2_patch_pc.assetlist ^
+    --outputBundlePath game_v1_to_v2_patch.pak
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+In the next section, you'll test the new bundles with your project's release build.
+
+## Test the asset bundles
+
+You can test the asset bundles by simulating two user scenarios.
+
+Because your release build currently has the original v1 asset bundle, start by testing the scenario where a user has your project installed and applies the patch asset bundle containing just the updated assets. Copy the patch asset bundle to your release asset directory.
+
+```cmd
+copy C:\MyProject\AssetBundling\Bundles\game_v1_to_v2_patch_pc.pak ^
```

Comment:
It's probably worth talking about pak loading prioritization here, to help users debug. I don't remember the specifics, but there's a chance that you go through these steps and see no visual change, if the original paks end up taking priority.

The easiest way to handle this would be to not put the new pak file in the wildcard loaded directory, and instead in another directory, and add some C++ to the project that loads that new pak file, specifically after GameApplication initialization, where engine.pak and wildcard pak loading is done.

This is also good because, a more mature project will probably want to manage pak mounting timing more directly, to manage load times of the game and to make sure the right content is initialized at the right time.

Engine.pak is mounted in GameApplication via:
`m_archive->OpenPack("@products@", enginePakPath.Native());`
So a similar call (after getting access to the archive system) would be able to load paks from elsewhere.

Reply:
I understand, however, I think this suggestion is out of scope for this tutorial.